### ISSUE TITLE
TTS no longer works on Homepod Minis

### ISSUE THREAD

**Issue by lancercncs1** [postlund/pyatv#1704]
### Describe the bug

Until I had updated to the April version of Home Assistant last night, my TTS Automations on my Homepod Minis (using the Apple TV Beta integration) were working just fine.

Now they have just stopped working and I have no idea why. I've tried restarting the integration, restarting Home Assistant, power cycling my server, restarting my Apple TV..etc. I also tried going back to the previous version of Home Assistant before the update and that did not work either..

I can't manually trigger the automations either. All entities and devices show up as "enabled." Also, the logs show that the automations still get "triggered" except that the Homepod Minis do not do anything.

What could be going on here and how do I fix it? :(

Thanks!

### Error log

```log
<The logs say the following...


`Failed to connect
4:33:34 PM – (ERROR) Apple TV (custom integration) - message first occurred at 4:33:34 PM and shows up 3 times


Logger: custom_components.apple_tv
Source: custom_components/apple_tv/__init__.py:307
Integration: Apple TV
First occurred: 4:33:34 PM (3 occurrences)
Last logged: 4:33:34 PM
Failed to connect
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/asyncio/locks.py", line 413, in acquire
    await fut
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/config/custom_components/apple_tv/__init__.py", line 225, in _connect_loop
    await self._connect(conf)
  File "/config/custom_components/apple_tv/__init__.py", line 307, in _connect
    self.atv = await connect(conf, self.hass.loop, session=session)
  File "/usr/local/lib/python3.9/site-packages/pyatv/__init__.py", line 135, in connect
    await atv.connect()
  File "/usr/local/lib/python3.9/site-packages/pyatv/core/facade.py", line 565, in connect
    if await setup_data.connect():
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/mrp/__init__.py", line 934, in _connect
    await protocol.start()
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/mrp/protocol.py", line 140, in start
    self.device_info = await self.send_and_receive(
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/mrp/protocol.py", line 259, in send_and_receive
    return await self._receive(identifier, timeout)
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/mrp/protocol.py", line 271, in _receive
    await asyncio.wait_for(semaphore.acquire(), timeout)
  File "/usr/local/lib/python3.9/asyncio/tasks.py", line 494, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

==================================================================

Failed to set up remote control channel
4:33:29 PM – (ERROR) /usr/local/lib/python3.9/site-packages/pyatv/protocols/airplay/__init__.py - message first occurred at 4:33:29 PM and shows up 3 times


Logger: pyatv.protocols.airplay
Source: /usr/local/lib/python3.9/site-packages/pyatv/protocols/airplay/__init__.py:265
First occurred: 4:33:29 PM (3 occurrences)
Last logged: 4:33:29 PM

Failed to set up remote control channel
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/airplay/__init__.py", line 258, in _connect_rc
    await control.start(str(core.config.address), control_port, credentials)
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/airplay/remote_control.py", line 56, in start
    self.verifier = await verify_connection(credentials, self.connection)
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/airplay/auth/__init__.py", line 103, in verify_connection
    has_encryption_keys = await verifier.verify_credentials()
  File "/usr/local/lib/python3.9/site-packages/pyatv/protocols/airplay/auth/hap_transient.py", line 47, in verify_credentials
    await self.http.post("/pair-pin-start", headers=_AIRPLAY_HEADERS)
  File "/usr/local/lib/python3.9/site-packages/pyatv/support/http.py", line 359, in post
    return await self.send_and_receive(
  File "/usr/local/lib/python3.9/site-packages/pyatv/support/http.py", line 412, in send_and_receive
    raise exceptions.HttpError(
pyatv.exceptions.HttpError: HTTP/1.1 method POST failed with code 400: Bad Request`>


===========================================================
Media streaming is not possible with current configuration

Logger: custom_components.apple_tv.media_player
Source: custom_components/apple_tv/media_player.py:297
Integration: Apple TV
First occurred: 4:43:23 PM (3 occurrences)
Last logged: 5:30:00 PM

Media streaming is not possible with current configuration

```

### How to reproduce the bug?

I noticed it when I upgraded to the April (4.4) version of Home Assistant. 

### What is expected behavior?

My TTS automations should just run as they had been doing before. 

### Operating System

HASS OS

### Python

3.9

### pyatv

the pyatv version that comes with atv-beta 3.0.1

### Device

Apple TV 1K OS 15.4

### Additional context

I also noticed that the remote control feature for my Homepod Minis turned permanently off. And I cannot turn them on again. When I try to, the remote control feature immediately turns it itself off.

**Comment by lancercncs1** [comment]
Good news! I *THINK* I got this fixed. In case this helps someone else.. After removing and redownloading the Apple TV integration, I realized that the HomeKit speaker/TV access was inexplicably changed to "Only People Sharing this Home  (Should be set on "Everyone" or "Anyone on the Same Network").. Once I toggled that back to where it should be (do not know why it changed to begin with!), I was able to re-pair everything and get it working again!
